moveit_ros/moveit_servo/src/utils/common.cpp

```diff
@@ -42,7 +42,12 @@
 namespace
 {
 // The threshold above which `override_velocity_scaling_factor` will be used instead of computing the scaling from joint bounds.
-const double SCALING_OVERRIDE_THRESHOLD = 0.01;
+constexpr double SCALING_OVERRIDE_THRESHOLD = 0.01;
+// The angle threshold in radians below which a rotation will be considered the identity.
+constexpr double MIN_ANGLE_THRESHOLD = 1E-16;
+// A minimum of 3 points are used to help with interpolation when creating a trjectory messages.
+constexpr int NUM_POINTS_FOR_TRAJ_MSG = 3;
```

Comment:
maybe call it `MIN_POINTS_FOR_TRAJ_MSG`?

Reply:
Agree, much better name